<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgWorkflowCreator</class>
 <widget class="QMainWindow" name="DlgWorkflowCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing Workflow Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_1">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="canvasTabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mToolbar">
   <property name="windowTitle">
    <string>Navigation</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mActionOpen"/>
   <addaction name="mActionSave"/>
   <addaction name="separator"/>
   <addaction name="mActionRemoveStep"/>
   <addaction name="separator"/>
   <addaction name="mActionRun"/>
  </widget>
  <action name="mActionOpen">
   <property name="text">
    <string>Open workflow</string>
   </property>
   <property name="toolTip">
    <string>Open workflow (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mActionSave">
   <property name="text">
    <string>Save workflow</string>
   </property>
   <property name="toolTip">
    <string>Save workflow (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mActionRemoveStep">
   <property name="text">
    <string>Remove step</string>
   </property>
   <property name="toolTip">
    <string>Remove step</string>
   </property>
  </action>
  <action name="mActionRun">
   <property name="text">
    <string>Run workflow</string>
   </property>
   <property name="toolTip">
    <string>Run workflow (F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
